templates/jail.conf.erb

```diff
@@ -71,10 +71,15 @@ action_mw = %(banaction)s[name=%(__name__)s, port="%(port)s", protocol="%(protoc
 action_mwl = %(banaction)s[name=%(__name__)s, port="%(port)s", protocol="%(protocol)s", chain="%(chain)s"]
                %(mta)s-whois-lines[name=%(__name__)s, dest="%(destemail)s", logpath=%(logpath)s, chain="%(chain)s", sendername="%(sendername)s"]
 
+<% if @slack_webhook %>
+action_with_slack_notification = %(banaction)s[name=%(__name__)s, bantime="%(bantime)s", port="%(port)s", protocol="%(protocol)s", chain="%(chain)s"]
+                                 slack[name=%(__name__)s]
+<% end %>
+
 # Choose default action.  To change, just override value of 'action' with the
 # interpolation to the chosen action shortcut (e.g.  action_mw, action_mwl, etc) in jail.local
 # globally (section [DEFAULT]) or per specific section
-action = %(<%= node['fail2ban']['action'] %>)s
+action = %(<%= @slack_webhook ? 'action_with_slack_notification' : node['fail2ban']['action'] %>)s
```

Comment:
See parent comment.  this overrides what's set in `node['fail2ban']['action']`.  Maybe that's intended behavior?

Reply:
I made it like this because it doesn't make sens to set the Slack webhook, but use another action. That's also why the slack action is only added to the file when `@slack_webhook` is set.